tests/flytekit/unit/configuration/configs/sample.yaml

```diff
@@ -11,3 +11,6 @@ storage:
     access-key: minio
     endpoint: http://localhost:30084
     secret-key: miniostorage
+images:
```

Comment:
I don't think this is supported at all on the flytectl side (and might never be since it's mostly responsible for registration time settings) but that's ok.  We can still support this.